test/TestGrains/StatsCollectorGrain.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Threading.Tasks;
+using Orleans;
+using Orleans.Placement;
+
+namespace UnitTests.Stats
+{
+    [PreferLocalPlacement]
+    public class StatsCollectorGrain : Grain, IStatsCollectorGrain
+    {
+        private long numStatsCalls = -1;
+        private long numMetricsCalls = -1;
+
+        public Task ReportMetricsCalled()
+        {
+            numStatsCalls++;
+            return TaskDone.Done;
+        }
+
+        public Task ReportStatsCalled()
+        {
+            numMetricsCalls++;
+            return TaskDone.Done;
+        }
+
+        public Task<long> GetReportMetricsCallCount()
```

Comment:
Took a look, and I still can't make sense why it was ever -1, but still, it does seem that 0 is the right value also according to how the test which uses it is asserting, so fixed